extensions/rich_text_components/rte-output-display.component.ts

```diff
@@ -184,6 +184,16 @@ export class RteOutputDisplayComponent implements AfterViewInit {
        * cycle and hence, we will still have the problem.
        */
       this.show = false;
+      const textNodes: Text[] = [];
```

Comment:
@ashutoshc8101 Could you please add some comments here to explain what's going on?

Also, if this is a temporary fix, maybe add a TODO + issue?

Reply:
@ashutoshc8101 you can add this to the paragraph above (lines 171-184). The text node we inserted outside angulars influence is causing some leaks. So we are manually removing the text nodes we attached.

Reply:
done